### PR TITLE
DX-535-Update the example in ExecCommand

### PR DESCRIPTION
**Pull request by shikhanansi**
Hi Mike,
I've made the changes to the ExecCommand section as per the request in https://ephocks.atlassian.net/browse/DX-535.
Please review the changes.
Regards,
Shikha

### CODE REVIEW

File: advanced/events.md [hunk 1]
```diff
@@ -130,13 +130,13 @@ tinymce.init({
...
 
-Gets fired after contents been inserted into the editor.
+Gets fired after the content has been inserted into the editor.
 
 #### Parameters
 * **content** `String` - The HTML content that was inserted into the editor.
 * **selection** `Boolean` - True/False if the content was inserted at selection or replaced all contents.
 
-Here is an example on log the content that got inserted into the editor.
+Here is an example of a content log that got inserted into the editor.
```
Comment (EkimChau) [inline advanced/events.md]: Here is a console log example when content has been inserted into the editor

File: advanced/events.md [hunk 2]
```diff
@@ -171,12 +171,12 @@ tinymce.init({
 
 ### PreProcess
 
-This event gets fired when the contents inside the editor is being serialized to a HTML string.
+This event gets fired when the contents inside the editor are being serialized to an HTML string.
```
Comment (EkimChau) [inline advanced/events.md]: revert to previous

File: advanced/events.md [hunk 3]
```diff
@@ -191,12 +191,12 @@ tinymce.init({
 
 ### PostProcess
 
-This event gets fired when the contents inside the editor have been serialized to a HTML string.
+This event gets fired when the contents inside the editor have been serialized to an HTML string.
```
Comment (EkimChau) [inline advanced/events.md]: revert to previous

File: advanced/events.md [hunk 4]
```diff
@@ -252,12 +252,12 @@ tinymce.init({
 
 ### Redo
 
-This event gets fired when redo is made by the user.
+This event gets fired when a request to redo the changes made by the Undo event is made by the user.
```
Comment (EkimChau) [inline advanced/events.md]: The previous version is simpler

File: advanced/events.md [hunk 5]
```diff
@@ -321,32 +321,34 @@ tinymce.init({
 
 ### ExecCommand
 
-This even is fired when a command like Bold/Italic etc is made by the editor.
+This event is fired when a command like Bold/Italic etc is made by the editor.
 
 #### Parameters
-* **content** `String` - The HTML content that's being extracted from the editor.
+* **content** `String` - The name of the command that was executed.
```
Comment (EkimChau) [inline advanced/events.md]: change **content** `String`
to **command** `String`

File: advanced/events.md (hunk 2 above)
Comment (techtangents) [reply advanced/events.md]: We will be using the "aytch" pronunciation.

File: advanced/events.md [hunk 6]
```diff
@@ -25,36 +25,36 @@ tinymce.init({
...
-| [MouseOut](https://developer.mozilla.org/en/docs/Web/Events/mouseout) | native | Fires when a element is no longer being hovered within the editor. |
-| [MouseEnter](https://developer.mozilla.org/en/docs/Web/Events/mouseenter) | native | Fires when mouse enters the editor. |
-| [MouseLeave](https://developer.mozilla.org/en/docs/Web/Events/mouseleave) | native | Fires when mouse leaves the editor. |
-| [KeyDown](https://developer.mozilla.org/en/docs/Web/Events/keydown) | native | Fires when a key is pressed within the the editor. |
-| [KeyPress](https://developer.mozilla.org/en/docs/Web/Events/keypress) | native | Fires when a key is pressed within the the editor. |
-| [KeyUp](https://developer.mozilla.org/en/docs/Web/Events/keyup) | native | Fires when a key is released within the the editor. |
+| [MouseOut](https://developer.mozilla.org/en/docs/Web/Events/mouseout) | native | Fires when an element is no longer being hovered within the editor. |
+| [MouseEnter](https://developer.mozilla.org/en/docs/Web/Events/mouseenter) | native | Fires when a mouse enters the editor. |
```
Comment (techtangents) [inline advanced/events.md]: the mouse

File: advanced/events.md (hunk 1 above)
Comment (techtangents) [reply advanced/events.md]: In the following example, the content is logged to the console in response to he SetContent event.

File: advanced/events.md (hunk 4 above)
Comment (techtangents) [reply advanced/events.md]: This event gets fired when a request to redo is made by the user.

File: advanced/events.md [hunk 7]
```diff
@@ -392,16 +394,16 @@ tinymce.on('AddEditor', function (e) {
 | Event       |  Description   |
 |-------------|----------------|
 | [AddEditor](#addeditor) | Fires when a new editor instance is added. |
-| [RemoveEditor](#removeeditor) | Fires when a editor instance is removed. |
+| [RemoveEditor](#removeeditor) | Fires when an editor instance is removed. |
 
 ### AddEditor
 
-This event gets fired when editor instances gets created and added to the EditorManager collection.
+This event gets fired when editor instances get created and added to the EditorManager collection.
```
Comment (techtangents) [inline advanced/events.md]: This event is fired when an editor instance is created and added to the EditorManager collection.

File: advanced/events.md (hunk 6 above)
Comment (shikhanansi) [reply advanced/events.md]: Done.

File: advanced/events.md (hunk 1 above)
Comment (shikhanansi) [reply advanced/events.md]: To keep the wording consistent with the other examples on this page, I have changed it to:
"Here is an example of the content logged to the console in response to the SetContent event."

File: advanced/events.md (hunk 5 above)
Comment (shikhanansi) [reply advanced/events.md]: Done

File: advanced/events.md (hunk 4 above)
Comment (shikhanansi) [reply advanced/events.md]: Changed as suggested by @techtangents .

File: advanced/events.md (hunk 7 above)
Comment (shikhanansi) [reply advanced/events.md]: Changed as suggested by @techtangents .

**Pull request merged**